bigtable/google/cloud/bigtable/client.py

```diff
@@ -76,8 +76,15 @@ def _make_data_stub(client):
     :returns: A gRPC stub object.
     """
     if client.emulator_host is None:
+        one_hundred_mb = 100 * 1024 * 1024
+        # NOTE: 'grpc.max_message_length' will be deprecated in the 1.1 release
```

Comment:
When something is deprecated it still works. More accurate language would be to say that `grpc.max_message_length` will no longer be _recognized_ in 1.1-and-later.